pathway-analysis/03-gene_set_variation_analysis.nb.html

```diff
@@ -1127,10 +1127,10 @@ <h2>Session Info</h2>
 <pre><code>R version 4.0.3 (2020-10-10)
 Platform: x86_64-pc-linux-gnu (64-bit)
-Running under: Ubuntu 20.04.1 LTS
+Running under: Ubuntu 20.04 LTS
```

Comment:
Curious what the story is here with GHA...

Reply:
Oh, I suppose this is probably actually a `rocker` thing.... always interesting when there is a downgrade though.